test/ObjectField_test.js

```diff
@@ -720,13 +720,18 @@ describe("ObjectField", () => {
       });
 
       expect(
-        node.querySelector(".form-group > .row > .col-xs-2 .btn-danger")
+        node.querySelector(
+          ".form-group > .form-additional > .form-additional-actions .btn-danger"
```

Comment:
@pieplu What is `.form-additional-actions`? I can't find it in the codebase.

Reply:
Nice catch, it's an "artefact" of my className proposal. I will fix that now...